ls1intum/Artemis
# [Chore] Changing score to the correct noun: points

### Checklist

### Motivation and Context


There is a naming problem in our codebase. The **max_score** in an exercise represents the absolute number of points a student can achieve in the exercise and the **score** in result means the relative percentage of all points a student achieved. **This double meaning of score is confusing**


### Description


* Rename exercise.max_score into points (database, server, client)
* Rename quiz_question.score into points (database, server, client)
* Check other places like translation strings that the correct word "points" is used when it concerns points and not score
* Check that in all assessment screens the entry in feedback is changed from score to points **(make it clear to tutors that they are awarding points)** 
IMPORTANT: **To not break anything in the codebase the changes in the code itself are kept to the minimum such that everything still works. I know that there are still a lot of places left in the code where maxScore or score is used but changing all this would break the scope of this PR dramatically**



### Steps for Testing


**There is not really much to check here except that everywhere where the meaning is points it should be written points and not score**

In particular check all the manual assessment views

### Test Coverage



**No changes in test coverage**

### Screenshots


![programing assesment](https://user-images.githubusercontent.com/29718932/106506479-15645380-64ca-11eb-89fe-1e7bfb3f5f7e.png)
![quizPoints](https://user-images.githubusercontent.com/29718932/106506483-172e1700-64ca-11eb-8a37-6a81e7f7c499.png)
![text](https://user-images.githubusercontent.com/29718932/106506486-17c6ad80-64ca-11eb-8154-1db02adfbb90.png)
